### ISSUE TITLE
Skip to main content does not work (Accessibility)

### ISSUE THREAD

**Issue by mbstacy** [culibraries/ir-scholar#56]
The skip to content link is ineffective because it places focus on the title of the page, which is 1 line up from the skip to content link. A skip link is valuable in that it bypasses redundant information across a site (e.g. banner options) and allows a user to get right to the main content of the page.
